gallery/src/components/NxTag/NxTagPage.tsx

```diff
@@ -0,0 +1,133 @@
+const NxTagNarrowExampleCode = require('!!raw-loader!./NxTagNarrowExample').default;
+
+const NxTagPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p className="nx-p">
+        There are two types of <code className="nx-code">NxTag</code>. The basic tags have a single state.
+        <code className="nx-code">NxSelectableTag</code> can be selected and deselected. It is up to the consumer
+        what that means in the context of the UI. The are many colors provided
+        for <code className="nx-code">NxTag</code> which can be specified via props.
+      </p>
+      <h3 className="nx-h3">Common Props</h3>
+      <NxTable>
+        <NxTableHead>
+          <NxTableRow>
+            <NxTableCell>Prop</NxTableCell>
+            <NxTableCell>Type</NxTableCell>
+            <NxTableCell>Required</NxTableCell>
+            <NxTableCell>Default</NxTableCell>
+            <NxTableCell>Details</NxTableCell>
+          </NxTableRow>
+        </NxTableHead>
+        <NxTableBody>
+          <NxTableRow>
+            <NxTableCell>color</NxTableCell>
+            <NxTableCell>
+              'light-blue' | 'purple' | 'pink' | 'blue' | 'red' | 'green' | 'orange' | 'yellow' | 'lime' | 'indigo'
+            </NxTableCell>
+            <NxTableCell>No</NxTableCell>
+            <NxTableCell></NxTableCell>
+            <NxTableCell>
+              If no <code className="nx-code">color</code> is specified then the default (dark blue with a grey
+              unselected state) colors are used.
+            </NxTableCell>
+          </NxTableRow>
+        </NxTableBody>
+      </NxTable>
+      <h3 className="nx-h3">Selectable Tag Props</h3>
+      <NxTable>
+        <NxTableHead>
+          <NxTableRow>
+            <NxTableCell>Prop</NxTableCell>
+            <NxTableCell>Type</NxTableCell>
+            <NxTableCell>Required</NxTableCell>
+            <NxTableCell>Default</NxTableCell>
+            <NxTableCell>Details</NxTableCell>
+          </NxTableRow>
+        </NxTableHead>
+        <NxTableBody>
+          <NxTableRow>
+            <NxTableCell>onSelect</NxTableCell>
+            <NxTableCell>
+              Function
+            </NxTableCell>
+            <NxTableCell>No</NxTableCell>
+            <NxTableCell></NxTableCell>
+            <NxTableCell>
+              Callback that changes the value of <code className="nx-code">selected</code> below between true
+              (selected) and false (unselected).
+            </NxTableCell>
+          </NxTableRow>
+          <NxTableRow>
+            <NxTableCell>selected</NxTableCell>
+            <NxTableCell>
+              Boolean
+            </NxTableCell>
+            <NxTableCell>No</NxTableCell>
+            <NxTableCell></NxTableCell>
+            <NxTableCell>
+              Boolean for the selected/unselected state of a <code className="nx-code">NxSelectableTag</code>
+            </NxTableCell>
+          </NxTableRow>
+          <NxTableRow>
```

Comment:
This row should be in the Common Props table.

Reply:
> This row should be in the Common Props table.

https://github.com/sonatype/sonatype-react-shared-components/pull/290/commits/653cc4414867a490d1bd26e06ba7a19684a89e4d